### DISCUSSION TITLE
Nuxt Internal Server Error after vercel deployment

### DISCUSSION THREAD

**Discussion by jonaferm** [nuxt/nuxt#24572] [Questions (Nuxt 2)]
Hello,

i have an issue with my deployed page in vercel.
The deployment sucess but I get the error "Nuxt Internal Server Error" when i open the page.

![image](https://github.com/nuxt/nuxt/assets/6004804/c47be6fe-77fb-4723-b813-544e3786f59d)

Here is my config file 

export default {
    
    components: true,
    //ssr : false,
    head: {
        titleTemplate: "%s",
        htmlAttrs: {
            lang: "en"
        },
        bodyAttrs: {
            class: ["my-style"]
        },
        meta: [{
            charset: "utf-8",
        }]
    },
    router: {
        prefetchLinks: false,
    },

        plugins:[
            /*'~/plugins/maps.client',*/
            '~/plugins/dataApi', 
            /*'~/plugins/daily', */
            '~/plugins/auth.client',
            { src: '~/plugins/gallery', mode: 'client' },
            /*'~/plugins/vCalendar.client',*/
            '~/plugins/stripe.client',
            '~/plugins/vue-scrollama.js',
            { src: '~/plugins/vercel.js', mode: 'client' },
        ],
        //the object represent the module config
        modules:[
            '~/modules/auth',
            '~/modules/algolia',
            '~/modules/cloudinary',
            '@nuxtjs/cloudinary',
            '~/modules/stripe',
            '@nuxtjs/axios',
            ['nuxt-mail', {
              message: {
                to: 'jon-live@hotmail.fr',
              },
              smtp: {
                host: 'smtp.mailtrap.io',
                port: 2525,
                auth: {
                  user: 'blabla',
                  pass: 'blabla'
                },
              },
            }],
        ],
        buildModules:[
            '@nuxtjs/tailwindcss',
            '@nuxt/image',
            'nuxt-gsap-module',
        ], //'vue-plausible'
        /*plausible: { // Use as fallback if no runtime config is available at runtime
            domain: 'https://localhost:3000/admin',
            trackLocalhost: true,
          },*/
        cloudinary:{ //The Cloudinary module gets its config from a property called cloudinary
            cloudName: 'soft-power',
        },
        image: {
            cloudinary: {
              baseURL: 'https://res.cloudinary.com/soft-power/image/upload/'
            }
        },
        
        css: [
            '~/assets/sass/app.scss',
        ],

        build: {
            extractCSS: true,
            loaders: {
                //only files zero bytes will be inlined
                limit: 0,
            },
            transpile: ["vue-picture-swipe"],
            babel: {
                plugins: [
                  ['@babel/plugin-proposal-class-properties', { loose: true }],
                  ['@babel/plugin-proposal-private-methods', { loose: true }],
                  ['@babel/plugin-proposal-private-property-in-object', { loose: true }]
                ],
              },
        },

        gsap: {
            extraPlugins: {
                scrollTrigger: true,
                scrollTo: true
            }
        },

        publicRuntimeConfig:{

            rootUrl : process.env.NODE_ENV === 'production' 
                ? 'https://hanna-photos.lu'
                : 'http://localhost:3000',
            /*plausible: {
                domain: process.env.PLAUSIBLE_DOMAIN,
                apiHost: process.env.PLAUSIBLE_API_HOST
              },*/
            auth:{
                cookieName: 'idToken',
                clientId: 'blabla'
            },
            algolia:{
                appId: 'blabla',
                //api key for search only
                key: 'blabla',
            },
            cloudinary:{
                apiKey: 'blabla',

            },
            stripe:{
                key: "blabla",
            },
        },
        //Only exposed Server Side :
        privateRuntimeConfig:{
            algolia:{
                appId: 'blabla',
                //api key for search and addobject
                key: 'blabla',
            },
            cloudinary:{
                apiSecret: 'blabla',
            },
            stripe:{ //.env
                secretKey: process.env.STRIPE_SECRET_KEY,
            },
        },

        target: 'static'

}

When i was building with this version i had no issue, but this is deprecated 

{
    "version": 2,
    "builds": [
      {
        "src": "nuxt.config.js",
        "use": "@nuxtjs/vercel-builder@0.22.1",
        "config" :{
          "serverFiles":["modules/**","utils/**"]
        }
      }
    ]
  }

thank you for your help !

**Comment by MarvinKweyu** [comment]
@jonaferm 
Install `node-fetch-native`

```bash
npm i  node-fetch-native
```
